test/test_distributed_models.jl

```diff
@@ -361,7 +361,11 @@ function test_triply_periodic_halo_communication_with_411_ranks(halo)
         @test all(south_halo(field, include_corners=false) .== arch.local_rank)
         @test all(top_halo(field, include_corners=false) .== arch.local_rank)
         @test all(bottom_halo(field, include_corners=false) .== arch.local_rank)
-    end
+
+        @test all(north_halo(field, include_corners=false) .== arch.connectivity.north)
+        @test all(south_halo(field, include_corners=false) .== arch.connectivity.south)
```

Comment:
@francispoulin I think these tests might be wrong? In this test there is only x decomposition so we expect the y halos (north and south) to just get filled with the value of the local rank (reflected in the tests a few lines above).

I think if you remove these two tests then the cpu distributed tests should all pass.

Reply:
Thanks a lot @ali-ramadhan , I will give it a try